pkg/flow/logs.go

```diff
@@ -21,6 +21,49 @@ type logMessage struct {
 	ns  *ent.Namespace
 	wf  *wfData
 	in  *ent.Instance
+	tag map[string]string
+}
+
+type taggedInstance interface {
+	tags() map[string]string
+	instance() *ent.Instance
+	//addTag(t, v string)
+}
+
+type taggedNamespace interface {
```

Comment:
I think we need to delete these interfaces and just pass both of these arguments in separately. That's should make resolving the merge conflicts easier.